charts/incubator/blocky/questions.yaml

```diff
@@ -622,6 +621,53 @@ questions:
                 schema:
                   type: string
                   default: ""
+              - variable: forward
+                label: Forward
+                schema:
+                  type: dict
+                  additional_attrs: true
+                  attrs:
+                    - variable: enabled
+                      label: Enabled
+                      schema:
+                        type: boolean
+                        default: false
+                    - variable: primary
+                      label: Primary
+                      schema:
+                        type: string
+                        show_if: [["enabled", "=", "true"]]
+                        default: tls://1.1.1.1
+                    - variable: secondary
+                      label: Secondary
+                      schema:
+                        type: string
+                        show_if: [["enabled", "=", "true"]]
+                        default: tls://1.0.0.1
+                    - variable: options
+                      label: Options
+                      schema:
+                        type: list
+                        show_if: [["enabled", "=", "true"]]
+                        default: []
+                        items:
+                          - variable: optionsEntry
+                            label: Options Entry
+                            schema:
+                              type: dict
+                              additional_attrs: true
+                              attrs:
+                                - variable: name
+                                  label: Name
+                                  schema:
+                                    type: string
+                                    default: ""
+                                - variable: value
+                                  label: Value
+                                  schema:
+                                    type: string
+                                    default: ""
+
```

Comment:
Why are you readding this?
I explained already that it's removed because it's not actually going to be used from the GUI.
(no forwardable traffic is getting to k8s-gateway, as it's pre-filtered)

Reply:
Didn't notice it was removed, thought the forward key added later..

Removed.